packages/terra-image/src/Image.jsx

```diff
@@ -23,6 +23,10 @@ const propTypes = {
    * The text content that specifies an alternative text for an image.
    */
   alt: PropTypes.string.isRequired,
+  /**
+   * An image which will be displayed during loading and in case of src load failure
```

Comment:
update this to include that this is the "src string of the image"

currently it makes me thing that I should pass in an Image node and not a string

Reply:
How do you feel about making it more specific as well (placeholderSrc)? I vote for this approach as some of the html nodes do have a placeholder attribute.

Reply:
+1